src/puzzlepieces/settings.py

```diff
@@ -27,7 +27,8 @@
 SECRET_KEY = os.getenv("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = int(os.environ.get("DEBUG", default=0))
+DEBUG = False
```

Comment:
shouldn't this stay an environment variable so you can set it differently depending if you're running a development machine vs production?

Reply:
Yes, as long as you can teach me how we do this in our dev/prod environment. Let's discuss, when we're all fresh, what we can do to get the current server better. That uswgi work will be key I think. When we switch over to run the current server on prod rather than dev, will we need to export / import SQL because we'll have a different mysql instance?